### DISCUSSION TITLE
filtering and root node

### DISCUSSION THREAD

**Discussion by autoca** [freeplane/freeplane#634] [Help] [answered]
Hi all,
it sometimes happens that when I filter out all ancestors of a certain level then:
(1) the root node does not get filtered out but is still displayed
(2) the other still visible part of the map visually overlaps (all of a sudden) with the root node

Is this happening to others too?
Is there a setting to avoid that or - even better - to also hide the root node as well? Maybe I overlooked it?

Regards, autoca

**Comment by macmarrum** [comment, marked as answer]
Root is never hidden. It's how Freeplane works.

Regarding (2), this sounds like a bug. Would you be able to create a report, including a sample mindmap and steps to reproduce the issue?

**Comment by euu2021** [comment]
There is a new feature, called Jump-In, that can be used to do exactly that. It still wasn't released in the stable version, though.
